common/persistence/nosql/nosql_execution_store_util_test.go

```diff
@@ -414,6 +415,60 @@ func TestPrepareNoSQLTasksForWorkflowTxn(t *testing.T) {
 	}
 }
 
+func TestPrepareTransferTasksForWorkflowTxn(t *testing.T) {
+	mockCtrl := gomock.NewController(t)
+	defer mockCtrl.Finish()
+
+	mockDB := nosqlplugin.NewMockDB(mockCtrl)
+	store := newTestNosqlExecutionStore(mockDB, log.NewNoop())
```

Comment:
create the mocks and the store object per test case inside t.Run to have full isolation